package.json

```diff
@@ -1,83 +1,85 @@
 {
   "name": "@leancodepl/contractsgenerator-typescript",
-  "description": "LeanCode Core Library TypeScript Contracts Generator",
-  "version": "0.5.0-alpha.0",
-  "author": "LeanCode",
-  "license": "APACHE-2.0",
-  "main": "lib/index.js",
-  "types": "lib/index.d.ts",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/leancodepl/contractsgenerator-typescript"
-  },
-  "bugs": {
-    "url": "https://github.com/leancodepl/contractsgenerator-typescript/issues"
-  },
-  "homepage": "https://github.com/leancodepl/contractsgenerator-typescript#readme",
-  "bin": {
-    "ts-generator": "lib/index.js"
-  },
-  "keywords": [
-    "contracts",
-    "leancode"
-  ],
-  "directories": {
-    "lib": "lib",
-    "test": "__tests__"
-  },
-  "files": [
-    "lib"
-  ],
+  "version": "0.0.0",
+  "license": "MIT",
   "scripts": {
-    "build": "rollup --config rollup.config.js",
-    "test": "jest --selectProjects generator eslint",
-    "test:watch": "jest --watch --selectProjects generator",
-    "test:all": "jest",
-    "lint": "eslint --fix src/**/*.{js,jsx,ts,tsx}",
-    "proto": "npm-run-all -s proto:download proto:generate proto:definitions",
-    "proto:download": "download -o src/protocol --filename contracts.proto https://raw.githubusercontent.com/leancodepl/contractsgenerator/main/contracts.proto",
-    "proto:generate": "pbjs -t static-module -w commonjs -o src/protocol/index.js src/protocol/contracts.proto --no-create --no-encode --lint \"eslint-disable prettier/prettier,no-undef,no-prototype-builtins,no-redeclare\"",
-    "proto:definitions": "pbts -o src/protocol/index.d.ts src/protocol/index.js"
+    "prepare": "chmod +x ./node_modules/husky/lib/bin.js && husky install && nx g @nx-dotnet/core:restore"
```

Comment:
I'm the one who added `chmod +x ...` but after further testing I think it's not necessary and should be remove (the same should be done in corelibrary)

Reply:
I was also thinking about that - it will break on windows